.github/workflows/assets/bootstrap/argocd-cmp-3.5/kustomization.yaml

```
@@ -3,7 +3,7 @@ kind: Kustomization
 namespace: argocd
 
 resources:
```

actionlint (1.7.12)

[error] unexpected key "resources" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)